packages/collector/test/tracing/messaging/bull/test.js

```diff
@@ -53,6 +54,44 @@ mochaSuiteFn('tracing/messaging/bull', function () {
   globalAgent.setUpCleanUpHooks();
   const agentControls = globalAgent.instance;
 
+  describe('allowRootExitSpan', function () {
+    let controls;
+
+    before(async () => {
+      controls = new ProcessControls({
+        useGlobalAgent: true,
+        appPath: path.join(__dirname, 'allowRootExitSpanApp'),
+        env: {
+          REDIS_SERVER: `redis://${process.env.REDIS}`,
+          BULL_QUEUE_NAME: queueName,
+          BULL_JOB_NAME: 'steve'
+        }
+      });
+
+      await controls.start(null, null, true);
+    });
+
+    beforeEach(async () => {
+      await agentControls.clearReceivedTraceData();
+    });
+
+    it('must trace', async function () {
+      await retry(async () => {
+        const spans = await agentControls.getSpans();
+
+        // TODO: all other bull tests also produce a huge number of spans
+        //       https://jsw.ibm.com/browse/INSTA-15029
+        expect(spans.length).to.be.eql(6);
```

Comment:
After the fix, now we only have 1 exit span. That should be our actual expectation here!